backend/core/src/listings/listings.service.ts

```diff
@@ -78,8 +96,12 @@ export class ListingsService {
       // (WHERE params are the values passed to andWhere() that TypeORM escapes
       // and substitues for the `:paramName` placeholders in the WHERE clause.)
       .setParameters(innerFilteredQuery.getParameters())
-      .orderBy(getOrderByCondition())
+      .orderBy(getOrderByCondition(params))
+      // Order by unitSummary.unitType.numBedrooms and units.maxOccupancy so that, for a
+      // given listing, its unitSummary's or units are sorted from lowest to highest
```

Comment:
nit: `unitSummary's` should be `unitSummaries` right?

Reply:
Yeah good call!